test/server/cards/01_SOR/leaders/CassianAndorDedicatedToTheRebellion.spec.ts

```diff
@@ -0,0 +1,130 @@
+describe('Cassian Andor, Dedicated to the Rebellion', function() {
+    integration(function(contextRef) {
+        describe('Cassian Andor\'s leader ability', function() {
+            it('should draw a card aftering dealing 3 damage to an enemy base with an attack', function() {
+                contextRef.setupTest({
+                    phase: 'action',
+                    player1: {
+                        hand: ['daring-raid', 'daring-raid'],
+                        deck: ['k2so#cassians-counterpart', 'open-fire'],
+                        spaceArena: ['green-squadron-awing'],
+                        leader: { card: 'cassian-andor#dedicated-to-the-rebellion', deployed: false },
```

Comment:
The default state is not deployed so if you want you can just write `leader: 'cassian-andor#dedicated-to-the-rebellion'`. Having it be explicit is also fine though

Reply:
I'll remove it to be more consistent with the other tests.